src/externs/datatables.js

```diff
@@ -26,7 +27,27 @@ jQuery.prototype.DataTable = function(opt_params) {};
 DataTables.prototype.rows = function(opt_params) {};
 
 /**
- * @param {Object=} opt_params
+ * @param {number} params
  * @return {DataTables.Api}
  */
-DataTables.prototype.invalidate = function(opt_params) {};
+DataTables.prototype.button = function(params) {};
+
+/**
+ * @param {Object=} opt_params
+ */
+DataTables.Api.prototype.invalidate = function(opt_params) {};
+
+/**
+ * @param {Object=} opt_params
```

Comment:
Yes it should return DataTables.Api. But there will be issues when casting this data() into an array which is {!Array<!genotet.EdgeForTable>}. Maybe I could just add some annotation without casting coercively.